messages/off-chain-registry/message_manager.go

```diff
@@ -171,3 +171,11 @@ func (m *messageManager) SendMessage(signedMessage *warp.Message, destinationBlo
 func (m *messageManager) GetDestinationBlockchainID(unsignedMessage *warp.UnsignedMessage) (ids.ID, error) {
 	return unsignedMessage.SourceChainID, nil
 }
+
+func (m *messageManager) GetOriginSenderAddress(unsignedMessage *warp.UnsignedMessage) (common.Address, error) {
```

Comment:
We probably will want to loop back on the off-chain message support at a higher level either way after the next milestone, but should we parse the `unsignedMessage` here for now and return an error if the source address isn't the expected value?

Reply:
Good call. No need to assume the sender is what we expect here.